package.json

```diff
@@ -118,5 +118,9 @@
       "workshop-solutions/*",
       "slides/*"
     ]
+  },
+  "dependencies": {
+    "react": "^0.0.0-experimental-4ead6b530",
+    "react-dom": "^0.0.0-experimental-4ead6b530"
```

Comment:
why we need this?

Reply:
Because I was getting an error that I only know how to fix running `yarn add -W react@experimental react-dom@experimental`

Reply:
what was the error?

this could break the workshop

Reply:
I see. Is it better to remove concurrent mode (`useTransition`) or just move these installations to the local package "@workshop/13-useQueryLoader"?

Reply:
what is the errors?

Reply:
I remove the dependencies from root `package.json`, run `yarn --force` and it worked.